pkgs/applications/radio/yaac/default.nix

```diff
@@ -0,0 +1,34 @@
+{ makeWrapper, lib, stdenv, fetchurl, unzip, jre8 }:
+
+stdenv.mkDerivation rec {
+  version = "1.0-beta181";
+  pname = "YAAC";
+
+  src = fetchurl {
+    url = "https://www.ka2ddo.org/ka2ddo/YAAC.zip";
+    sha256 = "564071bb51fb4a91850ff53ff7d0c9477c1715d7b11eed9e97c89ead9df7332b";
+  };
+
+  nativeBuildInputs = [ unzip makeWrapper ];
+
+  unpackPhase = "unzip $src -d $out";
+
+  # TODO: Make Desktop Icon
+
+  installPhase = ''
```

Comment:
Remember to call `runHook` when overwriting a phase.
https://nixos.org/manual/nixpkgs/stable/#sec-stdenv-phases

Reply:
Updated in latest.